### ISSUE TITLE
Installation Error: 12157 

### ISSUE THREAD

**Issue by Ahirlabs** [aptana/studio3#538]
![error](https://user-images.githubusercontent.com/34760343/133802329-dc4f1a9b-c163-4ae1-9307-e387a9dfdc22.PNG)

**Comment by bghong** [comment]
Yes I have the same issue. I switched off my firewall re-installed but still had similar issue, could not acquire the file. I couldn't upload the pic.

**Comment by ktwrd** [comment]
Have you checked if the file is accessible on your browser?
